# Week 3: RNA-Seq: Methods

## Topics

This week's assignments will guide you through the following topics:
* Sample Collection 
* RNA-Sequencing

## Readings

Please read the following:
* **Replication Paper**: Ramaker et al. *Genome Medicine* (2017) {cite}`ramaker_post-mortem_2017` [[URL](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-017-0458-5)]; [[PDF](https://github.com/ShanEllis/capstone-genetics-domain/raw/master/papers/main-paper.pdf)]
    * Methods: beginning - stop at sequencing data analysis 
* RNA-Seq: Wang et al. *Nature Reviews Genetics* (2009) {cite}`wang_rna-seq_2009`; [[URL](https://www.nature.com/articles/nrg2484)]; [[PDF](https://github.com/ShanEllis/capstone-genetics-domain/raw/master/papers/week3/RNA-Seq.pdf)]


## Weekly Questions

Answer the following questions on this week's [Google Form](https://docs.google.com/forms/d/e/1FAIpQLSfhPvVvQMjpkifyTQiQ-mARpNT38X_sdrVIoaWbIY3kiPuvqw/viewform?usp=sf_link).
 
## Possible Discussion Topics

- What's all this terminology?! (exonic, intronic, splice junction, expression levels, polyA tags, paired-end sequencing, deep sequencing, RNA fragmentation, 5' and 3', "background")
- RNA-Seq Review
    - Steps in the process
    - To discuss: "RNA-Seq has very low, if any, background signal because DNA sequences can been unambiguously mapped to unique regions of the genome." 
    - What's happened since this technology came out in 2009?
- Figure review

## Section Materials

- [Notes from discussion](https://shanellis.github.io/capstone-genetics-domain/notes/03_fa20.pdf)

## Student Q&A

Student responses on the Google Form: [Summary](https://shanellis.github.io/capstone-genetics-domain/responses/03.html)

Q1: **What does "background" mean when talking about RNA-Sequencing and microarrays?**  
A1: Because microarrays involve hybridization and fluorescence, there will always be background levels of fluorescence when collecting the data. Thus, microarrays give an analog output - we're looking at the relative intensity of something fluorescent to background levels and using that to determine expression levels. The stronger something fluoresces, the higher its expression. But, sometimes background in these experiments can be "noisy" (show up stronger in some samples) and can differ between samples. On the other hand, RNA-Seq provides a digital output. We can count explicitly how many reads align at a given position. There is no "background" against which we have to compare to collect these expression estimates. This makes RNA-Seq output a lot easier to quantify and a lot more accurate, relative to the output from microarrays. 
