# Week 1: Introduction to Human Genetics

## Topics

This week's assignments will guide you through the following topics:
* Human Genetics
* The Central Dogma of Genetics

## Reading

Please read the following before our first Wednesday discussion:
* GWAS: Resnick B. *Vox* (2018) {cite}`resnick_fascinating_2018` [[URL](https://www.vox.com/science-and-health/2018/8/23/17527708/genetics-genome-sequencing-gwas-polygenic-risk-score)]

## Weekly Questions

Weekly questions will start week 2. There is nothing for you to complete other than the reading this first week.

## Section Materials

- [slides used in lecture](https://docs.google.com/presentation/d/1DvBer_jL7_HI1GRQ_CT0s226Egxjw5jV9_GKiWS_Tyo/edit?usp=sharing) - requires use of UC login/gmail account

## Student Q&A

A brief summary of commonly-asked questions about the readings:

Q1: **What's this idea of *missing heritability*?**  
A1: So, missing heritability is a term that refers to how much of a trait/disease we can experimentally determine is accounted for by our genetics vs how much of the trait/disease we *understand* genetically. For example, we can do twin studies and determine that 80\% of a trait (like height, for example) is attributed to genetics ....whereas, 20\% may be environmental (diet, etc.). *However*, we may only know/have identified a handful of specific genes and mutations that explain height. The *known* mutations may describe 45\% of the variation we see in height in the population. This means that only 45\% can be explained by what we *know*.....and the 35\% ($80-45$) remaining percent is "missing". We *know* it exists and is attributable to genetics....we just don't know specifically (yet) which mutations/genes are involved in that remaining 35\%.

Q2: **What exactly is a SNP?**  
A2: A SNP (Single Nucleotide Polymorphism) is a single base (T,C,G or A) in the human genome that differs between people. To be considered a SNP, the alternative allele must show up in at least 1\% of the population.

Q3: **What doe the Manhattan Plot plot?**  
A3: Manhattan plots are used alllllll over genetics...but nowhere else, so totally reasonable you wouldn't be familiar yet. x-axis is chromosome position. y-axis is -log10pval. The y-axis is on this scale so that small p-values (strongest associations) are the peaks on the plot while things that aren't associated fall close to the axis (0). Each point on the plot represents a SNP. So, for each SNP, the position in the genome is on the x-axis and how significantly associated that SNP is is on the y-axis. We see peaks b/c there is correlation within the genome. Take a look at the word "linkage disequilibrium" to get a better understanding. If still unclear, we should discuss more in discussion!




